docs/pages/access-controls/login-rules/reference.mdx

```diff
@@ -42,13 +42,13 @@ to remain populated. For example this configuration will keep the `groups` trait
 
 Here is an example Login Rule using a `traits_map` which implements the
 following rules:
-1. Every user with the `groups: devs` trait should receive an extra trait
+- Every user with the `groups: devs` trait should receive an extra trait
    `access: [staging]`.
-2. Every user with the `groups: admins` trait should receive an extra trait
+- Every user with the `groups: admins` trait should receive an extra trait
    `access: [staging, prod]`.
-3. Every user should receive a `logins` trait with the value of their incoming
+- Every user should receive a `logins` trait with the value of their incoming
    `username` trait converted to lowercase.
-4. All traits other than `groups`, `logins`, and `access` should be filtered
+- All traits other than `groups`, `logins`, and `access` should be filtered
```

Comment:
This is probably a good change, but it's worth double-checking; does the order of rule application matter in general and/or this example? If so, it might make sense to leave these numbered, to better enforce the concept of rule ordering.

Reply:
From reading the context on the page, the order in which the examples rules are listed isn't significant. Nic verified the unordered list is fine.